internal/pkg/cli/command/target/target.go

```diff
@@ -52,7 +38,15 @@ func NewTargetCmd() *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "target <flags>",
 		Short: "Set context for the CLI",
-		Long:  targetHelp,
+		Long: help.Long(`
+			Many API calls take place in the context of a specific project.
+			When using the CLI interactively (i.e. via the oauth2 authentication flow) you
+			should use this command to set the current project context for the CLI.
+
+			For automation use cases relying on API Keys for authentication, there's no need
+			to specify a project context as the API Key is already associated with a specific
```

Comment:
```suggestion
			For automation use cases relying on API keys for authentication, there's no need
			to specify a project context as the API key is already associated with a specific
```

In our larger body of docs, we don't typically capitalize resource names like Index or Collection. So I don't see why we would capitalize Keys here.

Reply:
Another good catch, I had tried to un-capitalize where I could, but missed a few. 👍